### ISSUE TITLE
Normdaten: Nicht-relevante Autorensätze

### ISSUE THREAD

**Issue by IxTheoKm** [ubtue/tuefind#2237]
Ein Normsatz, der fälschlich die individualisierenden Merkmale zweier verschiedener Personen enthielt und mit dem IxTheo-Titelsätze verknüpft waren, wurde korrigiert:
https://ixtheo.de/Authority/720899435

Die Titel sind nun mit einem anderen Normsatz korrekt verknüpft:
https://ixtheo.de/Authority/792711211

PPN 720899435 ist nicht relevant, aber weiterhin im IxTheo vorhanden. 
Wie verschwindet er wieder aus der Datenbank? Bei einem neuen Komplettabzug?

(Vgl. https://github.com/ubtue/tuefind/issues/1573)

**Comment by mtrojan-ub** [comment]
Ja, im Moment verschwindet er erst bei einem neuen Komplettabzug. Es gibt momentan keine Löschlisten speziell für Normdaten, und auch kein Programm o.Ä. das Normdaten ohne verknüpfte Titel aus unseren Pseudokomplettabzügen entfernt.

**Comment by mtrojan-ub** [comment]
Meine letzte Aussage war leider nicht korrekt.

In den vorhandenen Löschlisten gibt es laut Definition einen Indikator der kennzeichnet ob es sich um einen Titelsatz, Personensatz o.Ä. handelt. Siehe: https://wiki.k10plus.de/display/K10PLUS/Allgemeines+zur+Datenbereitstellung
(Sekundärkorrekturen => Ausführliche Informationen zu den Sekundärkorrekturen => Löschungen)

Bislang wenden wir alle PPNs mit Indikator = A-E auf die Titeldaten an. Das wäre vermutlich falsch und wir müssten nur A auf Titeldaten und B-E auf Normdaten anwenden.

Trotzdem hätte das in diesem Fall nichts gebracht, da die genannte Normdaten-PPN in keiner Löschliste vorhanden war (und Stichproben der anderen gelisteten Normdaten-PPNs konnte ich nicht im IxTheo finden).

Stattdessen wurde in der WA-MARCk10comb-sekkor-220715.tar.gz eine Korrektur des Datensatzes ausgeliefert. Hätte man in diesem Datensatz stattdessen erkennen können, dass es sich um einen zu löschenden Datensatz handelt? (Laut Doku beziehen sich die Sekkor-Dateien aber nur auf Korrekturen, nicht auf Löschungen).

**Comment by IxTheoKm** [comment]
> Trotzdem hätte das in diesem Fall nichts gebracht, da die genannte Normdaten-PPN in keiner Löschliste vorhanden war (und Stichproben der anderen gelisteten Normdaten-PPNs konnte ich nicht im IxTheo finden).
> 
> Stattdessen wurde in der WA-MARCk10comb-sekkor-220715.tar.gz eine Korrektur des Datensatzes ausgeliefert. Hätte man in diesem Datensatz stattdessen erkennen können, dass es sich um einen zu löschenden Datensatz handelt? (Laut Doku beziehen sich die Sekkor-Dateien aber nur auf Korrekturen, nicht auf Löschungen).

Es wurde in diesem Fall kein Personennormsatz gelöscht. Der wurde nur korrigiert. 
Die damit fälschlich verknüpften theologischen Titel wurden an einen anderen Personennormsatz umverknüpft. 

Es kann auch Fälle geben, in denen theologische Titel an andere Personennormsätzen umverknüpft werden, weil sie bisher an falschen gleichnamigen Personen hingen. Auch dann wäre der ursprüngliche, nicht korrigierte Normsatz nicht relevant für den IxTheo.

Momentan scheint es mir dafür nur zwei Lösungen zu geben:

1. Wie bisher auf einen neuen Komplettabzug warten, oder
2. Löschen von Normsätzen, die nicht mit Titeln verknüpft sind.

Bei Punkt zwei ist mir aber unklar, was https://github.com/ubtue/tuefind/issues/1573#issue-945275912 dann dafür bedeutet.
